api/lib/opentelemetry/trace/tracer.rb

```diff
@@ -31,22 +31,22 @@ def current_span
       #
       # With this helper:
       # OpenTelemetry.tracer.in_span('do-the-thing') do ... end
-      def in_span(name, sampler: nil, links: nil, recording_events: nil, kind: nil)
-        span = start_span(name, sampler: sampler, links: links, recording_events: recording_events, kind: kind)
+      def in_span(name, attributes: nil, links: nil, events: nil, resource: nil, span_id: nil, start_timestamp: nil)
+        span = start_span(name, attributes: attributes, links: links, events: events, resource: resource, span_id: span_id, start_timestamp: start_timestamp)
         with_span(span) { |s| yield s }
       end
 
       def with_span(span)
         Context.with(CONTEXT_SPAN_KEY, span) { |s| yield s }
       end
 
-      def start_root_span(name, sampler: nil, links: nil, recording_events: nil, kind: nil)
+      def start_root_span(name, attributes: nil, links: nil, events: nil, resource: nil, span_id: nil, start_timestamp: nil)
         raise ArgumentError if name.nil?
 
         Span.create_random
       end
 
-      def start_span(name, with_parent: nil, with_parent_context: nil, sampler: nil, links: nil, recording_events: nil, kind: nil)
+      def start_span(name, with_parent: nil, with_parent_context: nil, attributes: nil, links: nil, events: nil, resource: nil, span_id: nil, start_timestamp: nil)
```

Comment:
It seems like we might need a `end_timestamp` according to this comment: https://github.com/open-telemetry/opentelemetry-specification/pull/215#discussion_r315437851. This would apply to `in_span` and `start_root_span` as well.

Reply:
I think that comment misinterprets the RFC. The [relevant text](https://github.com/open-telemetry/rfcs/blob/8dd6e21/text/0002-remove-spandata.md#internal-details) only has the `end_timestamp` passed to `span.end()` (`span.finish` in our implementation).